torch/_inductor/ir.py

```diff
@@ -6444,7 +6444,19 @@ def create(cls, kernel, *args, **kwargs):  # type: ignore[no-untyped-def]
                 unflatten_args,
                 unbacked_bindings=unbacked_bindings,
             )
-
+        elif (not isinstance(example_output, tuple)) and (
+            not isinstance(example_output, list)
+        ):
+            packed = cls(
+                cls.tensor_to_layout(example_output),
+                kernel,
+                tensor_args,
+                non_tensor_args,
+                unflatten_args,
+                unbacked_bindings=unbacked_bindings,
+            )
+            packed.outputs = [packed]
```

Comment:
This is creating a reference cycle which isn't ideal.

Reply:
done